### PR TITLE
Update federalist-docs to use web design standards and not guides-style

### PR DESCRIPTION
**Pull request by wslack**
https://federalist.18f.gov/preview/18f/federalist-docs/uswds/

- [x] Update Footer
  - Inspiration: https://github.com/18F/18f.gsa.gov/blob/43726b189c08b3dc500110ed671e9dc89226c30c/_includes/navigation/footer-desktop.html

- [x] Check to see if USWDS are fixing the text size so pages have less white space
- [x] Update header, if desired, to have other menus

### CODE REVIEW

**Comment by wslack** [comment]
https://github.com/18F/web-design-standards/issues/1603 isn't happening yet but isn't a blocker

**Comment by wslack** [comment]
No longer WIP, @jmhooper or @brandocalrissian to review.

**Comment by wslack** [comment]
HOLD, found a bug.

**Comment by wslack** [comment]
ok, bug fixed. https://federalist.18f.gov/preview/18f/federalist-docs/uswds to review

**Comment by wslack** [comment]
Hmmm, @shawnbot it looks like this version doesn't work on mobile: compare it to https://standards.usa.gov/page-templates/docs/ (that's not from my footer work, right?)

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+<footer class="usa-footer usa-footer-medium" role="contentinfo">
+  <div class="usa-grid usa-footer-return-to-top">
+    <a href="#">Return to top</a>
+  </div>
+</footer>
```
Comment (wslack) [inline _includes/footer.html]: removed this content
`
  <div class="usa-footer-primary-section">
    <div class="usa-grid-full">
      <nav class="usa-footer-nav">
        <ul class="usa-unstyled-list">
          <li class="usa-width-one-fourth usa-footer-primary-content">
            <a class="usa-footer-primary-link" href="javascript:void(0);">Primary link</a>
          </li>
          <li class="usa-width-one-fourth usa-footer-primary-content">
            <a class="usa-footer-primary-link" href="javascript:void(0);">Permanently relevant</a>
          </li>
          <li class="usa-width-one-fourth usa-footer-primary-content">
            <a class="usa-footer-primary-link" href="javascript:void(0);">Easy to understand</a>
          </li>
          <li class="usa-width-one-fourth usa-footer-primary-content">
            <a class="usa-footer-primary-link" href="javascript:void(0);">Site policies (example)</a>
          </li>
        </ul>
      </nav>
    </div>
  </div>

  <div class="usa-footer-secondary_section">
    <div class="usa-grid">
      <div class="usa-footer-logo usa-width-one-half">
        Logo image
        <h3 class="usa-footer-logo-heading">Name of Agency</h3>
      </div>
      <div class="usa-footer-contact-links usa-width-one-half">
        <a class="usa-link-facebook" href="javascript:void(0);">
          <span>Facebook</span>
        </a>
        <a class="usa-link-twitter" href="javascript:void(0);">
          <span>Twitter</span>
        </a>
        <a class="usa-link-youtube" href="javascript:void(0);">
          <span>YouTube</span>
        </a>
        <a class="usa-link-rss" href="javascript:void(0);">
          <span>RSS</span>
        </a>
        <address>
          <h3 class="usa-footer-contact-heading">Agency Contact Center</h3>
          <p>(800) CALL-GOVT</p>
          <a href="mailto:info@agency.gov">info@agency.gov</a>
        </address>
      </div>
    </div>
  </div>`

**Comment by wslack** [comment]
@shawnbot ping with ❤️

**Comment by wslack** [comment]
`This build completed successfully.` but looks like we have the same problem: https://federalist.18f.gov/preview/18f/federalist-docs/uswds/

**Comment by wslack** [comment]
huh. Any chance this will impact others that want to add logos to a USWDS site, or is this special to us? This fix seems fine with me, would like @brandocalrissian or @jmhooper to check it though.

**Comment by jmhooper** [comment]
I'm looking at an issue in the menu on mobile:

Top of the page:

image

Bottom of the page:

image

**Comment by jmhooper** [comment]
@shawnbot: I was expecting the navigation items to appear in the menu on mobile, so that's why I thought there was an issue. If that behavior is intentional, then this LGTM.

**Comment by wslack** [comment]
:thinking: we've never had a menu nav like that available before in the previous page org.

I still like the nav on the left, but I wonder if we should add a local search option in the top nav. Doesn't have to happen now though.

However, https://federalist.18f.gov/preview/18f/federalist-docs/uswds/pages/how-federalist-works/ still looks weird because of the width issues - I pinged in https://github.com/18F/web-design-standards/issues/1603 about those.

**Comment by wslack** [comment]
Update: There is *no urgency* to this PR, so no rush on the design standards issue!

**Comment by wslack** [comment]
currently blocked by https://github.com/18F/federalist-docs/issues/45

**Comment by wslack** [comment]
This is no longer blocked.

**Pull request merged**